### DISCUSSION TITLE
--

### DISCUSSION THREAD

**Discussion by LilyEssence** [community/community#108528] [Pull Requests] [answered]
--

**Comment by spenserblack** [comment, marked as answer]
I think that's the intended behavior? `-draft:true` is negating the filter, like "*not* draft is true." Why not just `draft:false` instead of `-draft:true`?

See also this tip at the top of the linked docs:

> Use a minus (hyphen) symbol to exclude results that match a qualifier. For example, to ignore issues created by the "octocat" user, you'd use -author:octocat in your search.

******

Side note: I did notice that `-FILTER` negates *some* filters (like `-review:approved` gets PRs that are *not* approved), but the `-` seems to have no affect on other filters (`is:open` and `-is:open` both seem to show open issues). So it looks like the `-` doesn't apply to all filters.
